### DISCUSSION TITLE
Shouldnt .eslintignore be included in the globalDependencies?

### DISCUSSION THREAD

**Discussion by Kamahl19** [vercel/turborepo#8276] [Help] [answered]
### Summary

Shouldnt .eslintignore, .prettierignore and possibly other .ignore files be included in the globalDependencies? When we remove something from the .eslintignore file and run turbo command, how does it know the cache should change and lint command should run on this file?

**Comment by anthonyshew** [comment, marked as answer]
Yes, it would.

**Reply by Kamahl19** [reply]
I suppose it would be useful to mention in the docs or possibly add to the default global files, alongside package.json and turbo.json. 

What about the .gitignore? I suppose that doesnt need to be listed as a dependency because when i changes, the cache changes implicitly because files included in git changed, right?
